sympy/utilities/lambdify.py

```diff
@@ -88,22 +90,74 @@
     "Matrix": "array",
     "MutableDenseMatrix": "array",
     "ImmutableMatrix": "array",
+    "ImmutableDenseMatrix": "array"
+
+}
+
+# most of the functions of scipy is similar to numpy. Checked each functions.
```

Comment:
I don't think that there is any reason to have these doubled up. Can you do something like:

```python
NUMPY_TRANSLATIONS = {...}
SCIPY_TRANSLATIONS = {...}
SCIPY_TRANSLATIONS.update(NUMPY_TRANSLATIONS)
```

The SciPy namespace imports the numpy namspace so there shouldn't be a reason to write these duplicates.